webui/react/src/pages/TrialsComparison/Table/TrialTable.tsx

```diff
@@ -547,32 +548,36 @@ const TrialTable: React.FC<Props> = ({
       },
       fakeTotal,
     );
-  }, [settings.tableLimit, settings.tableOffset, trials.data.length]);
+  }, [settings?.tableLimit, settings?.tableOffset, trials.data.length]);
 
   return (
     <div className={css.base}>
-      <InteractiveTable<V1AugmentedTrial>
-        columns={columns}
-        containerRef={containerRef}
-        ContextMenu={ContextMenu}
-        dataSource={trials.data.slice(0, settings.tableLimit)}
-        interactiveColumns={false}
-        pagination={pagination}
-        rowClassName={highlights.rowClassName}
-        rowKey="trialId"
-        rowSelection={{
-          getCheckboxProps: () => ({ disabled: selectAllMatching }),
-          onChange: selectTrials,
-          preserveSelectedRowKeys: true,
-          selectedRowKeys: (selectAllMatching ? trials.ids : selectedTrials) as number[],
-        }}
-        scroll={{ x: 'max-content', y: '40vh' }}
-        settings={settings}
-        showSorterTooltip={false}
-        size="small"
-        updateSettings={updateSettings as UpdateSettings<InteractiveTableSettings>}
-        onRow={highlights.onRow}
-      />
+      {settings ? (
```

Comment:
I saw this pattern in several files. 
I think we can just wait until `settings` is loaded from db, then `settings` doesnt need to be null